app/controllers/concerns/hyrax/embargoes_controller_behavior.rb

```diff
@@ -24,23 +24,29 @@ def destroy
     end
 
     # Updates a batch of embargos
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength
     def update
       filter_docs_with_edit_access!
       copy_visibility = []
       copy_visibility = params[:embargoes].values.map { |h| h[:copy_visibility] } if params[:embargoes]
-      af_objects = Hyrax.custom_queries.find_many_by_alternate_ids(alternate_ids: batch, use_valkyrie: false)
-      af_objects.each do |curation_concern|
-        Hyrax::Actors::EmbargoActor.new(curation_concern).destroy
+      resources = Hyrax.custom_queries.find_many_by_alternate_ids(alternate_ids: batch, use_valkyrie: Hyrax.config.use_valkyrie?)
```

Comment:
i think we should consider just skipping all the `use_valkyrie?` stuff here. if this controller works with valkyrie and the impact is all internal, why not just use it and not present any other option?